### PR TITLE
chore(locale.de): update word definition

### PR DESCRIPTION
**Pull request by xDivisionByZerox**
This PR updates the word definitions for the german locale.

 Updated:
- adjective

Added:
- noun
- adverb
- verb
- preposition

### CODE REVIEW

**Review by pkuczynski — APPROVED** [review]
I can't judge `de` but I trust that you know what you are doing :)

**Review by import-brain — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]
I really like it.
Have you checked whether the adjective list allows for rebundling?
The page sells a product (writing/author class) so they may have interest in keeping it to their own.

File: src/locales/de/word/adjective.ts [hunk 1]
```diff
@@ -1,193 +1,1136 @@
+// source: https://buchkodex.de/blog/adjektive/
```
Comment (Shinigami92) [inline src/locales/de/word/adjective.ts:1]: @ST-DDT informed me about this PR and I'm of the opinion that we should not directly include this link in the source code but maybe just in the PR description (which is linked by the commit)
The adjectives here are not all from that list and in future we might add more to it and therefore just merge more and more together, so the source is not directly true anymore
So please remove it from this source file

Comment (Shinigami92) [reply src/locales/de/word/adjective.ts:1]: I'm ok with your suggestion

**Comment by pkuczynski** [comment]
I would consider Wikipedia more solid source of information...

**Review by pkuczynski — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**